integrations/tensorflow/build_tools/bazel/setup_local_llvm.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2021 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
```

Comment:
Add a comment with the script's purpose? Could just copy/paste from the PR description.